dalek/cleaners/resource_group_palo_alto_local_rulestack.go

```diff
@@ -0,0 +1,117 @@
+package cleaners
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonids"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/paloaltonetworks/2022-08-29/certificateobjectlocalrulestack"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/paloaltonetworks/2022-08-29/fqdnlistlocalrulestack"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/paloaltonetworks/2022-08-29/localrules"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/paloaltonetworks/2022-08-29/localrulestacks"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/paloaltonetworks/2022-08-29/prefixlistlocalrulestack"
+	"github.com/tombuildsstuff/azurerm-dalek/clients"
+)
+
+type PaloAltoLocalRulestackCleaner struct{}
+
+var _ ResourceGroupCleaner = PaloAltoLocalRulestackCleaner{}
```

Comment:
pedantic, but the others are private, so can we make this private:

```suggestion
type paloAltoLocalRulestackCleaner struct{}

var _ ResourceGroupCleaner = paloAltoLocalRulestackCleaner{}
```